### PR TITLE
Survival Knife sprite and sandbag layer fix

### PR DESCRIPTION
**Pull request by Bawhoppen**
Survival knife has new sprite courtesy of cheridan, and sandbag stacks are now a very tiny bit lower of a layer so they're not at the top of the locker when it is opened

### CODE REVIEW

**Comment by Core0verload** [comment]
meaningful pr names plz

**Comment by Bawhoppen** [comment]
this is literally the most accurate naming I could give it

**Comment by GunHog** [comment]
Sprite PR should include pictures of the sprite.

**Comment by Bawhoppen** [comment]
I intentionally bundled 2 things so I wouldn't get the sprite pr with no pics tag

**Comment by Jordie0608** [comment]
Well gee it happened anyway

**Comment by Bawhoppen** [comment]
reeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee

File: code/game/objects/items/stacks/sheets/mineral.dm [hunk 1]
```diff
@@ -63,6 +63,7 @@ var/global/list/datum/stack_recipe/sandstone_recipes = list ( \
 	icon_state = "sandbags"
 	singular_name = "sandbag"
 	force = 5
+	layer = 2.9005
```
Comment (phil235) [inline code/game/objects/items/stacks/sheets/mineral.dm]: You don't need that many decimals.

Comment (Bawhoppen) [inline code/game/objects/items/stacks/sheets/mineral.dm]: it doesn't hurt to have that many but what I intended moreso was more to be like 2.9995 but apparently I was tired

Comment (phil235) [inline code/game/objects/items/stacks/sheets/mineral.dm]: and try to use layer defines if possible.

**Comment by Supermichael777** [comment]
the tip of the knife doesn't look very natural. the backing stops short and the tip looks very fragile.
image of knife

**Comment by TechnoAlchemisto** [comment]
the handle looks a bit short as well IMO

**Comment by Jalleo** [comment]
It looks fine to me.

**Comment by Zilenan91** [comment]
I think the current knife sprite is better than what's up there tbh

**Comment by Bawhoppen** [comment]
the current one has the problem of being the same sprite as another way better weapon

**Comment by Zilenan91** [comment]
The combat knife that's in a crate that nobody ever orders because it got nerfed?

**Comment by Bawhoppen** [comment]
uh it actually got buffed???

**Comment by TechnoAlchemisto** [comment]
i hear if you add more question marks to your statements you become more and more passive aggressive

**Comment by Bawhoppen** [comment]
excuse me??????

File: code/game/objects/items/stacks/sheets/mineral.dm (hunk 1 above)
Comment (KorPhaeron) [inline code/game/objects/items/stacks/sheets/mineral.dm]: Why does it even need a higher layer at all?

**Comment by Bawhoppen** [comment]
cheridan my hero

File: code/game/objects/items/stacks/sheets/mineral.dm (hunk 1 above)
Comment (Bawhoppen) [inline code/game/objects/items/stacks/sheets/mineral.dm]: its a lower layer so its not at the top of the fucking mining lockers ree

**Pull request merged**